templates/abuse_report_location.conf.j2

```diff
@@ -1,6 +1,6 @@
 # Extracted from
-# https://github.com/matrix-org/mjolnir/blob/main/test/nginx.conf
-# Abuse reports should be sent to Mjolnir.
+# https://github.com/matrix-org/draupnir/blob/main/test/nginx.conf
```

Comment:
Hmm this reference i think wont have changed just because a migration.

Reply:
This URL is still wrong though

Reply:
Good catch, I changed the URL back to Mjolnir.